bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/handler/TelldusDevicesHandler.java

```diff
@@ -208,12 +210,21 @@ public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
                     }
                 }
             } catch (Exception e) {
-                logger.error("Failed to init bridge for {}", deviceId, e);
+                logger.warn("Failed to init device {}", deviceId, e);
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.HANDLER_INITIALIZING_ERROR);
             }
         } else {
-            updateStatus(ThingStatus.OFFLINE, bridgeStatusInfo.getStatusDetail());
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
```

Comment:
Is it needed?

Reply:
Yes, that is the case the bridge is going OFFLINE in case of communication error for example.

Reply:
If bridge will go offline, won't child handlers also?

Reply:
This is what `bridgeStatusChanged` does by default.
Here, `bridgeStatusChanged` is overwritten and so we have to handle the bridge OFFLINE case.
An alternative would be to call `super.bridgeStatusChanged(bridgeStatusInfo)` at this place but it is identical.
Here is the code of the method in BaseThingHandler:
```
    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
        if (bridgeStatusInfo.getStatus() == ThingStatus.ONLINE
                && getThing().getStatusInfo().getStatusDetail() == ThingStatusDetail.BRIDGE_OFFLINE) {
            updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE);
        } else if (bridgeStatusInfo.getStatus() == ThingStatus.OFFLINE) {
            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
        }
    }
```

Reply:
Thanks for the clarification, I didn't see this was overridden `bridgeStatusChanged`.